src/Components/ExistingTokenButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
+import { useHistory } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+
+export default function ExistingTokenButton({ setToken }) {
+  const [input, setInput] = useState('');
+  let history = useHistory();
+  const { register } = useForm();
```

Comment:
You can remove `react-hook-form` since you're only using it to register the token input but not to handle the submission of the form. If you want to keep it, you can remove the input state since `react-hook-form` will give you access to any input you register on submission.

Reply:
Thanks for pointing this out! We started using it then realized it wasn't necessary but forgot to delete it.